doc/reference/network_bridge.md

```diff
@@ -10,30 +10,15 @@ DHCP server and if desired also perform NAT for the bridge (this is the default.
 
 When a bridge is managed by LXD, configuration values under the `bridge` namespace can be used to configure it.
 
-Additionally, LXD can utilize a pre-existing Linux bridge. In this case, the bridge does not need to be created via
```

Comment:
Yep agreed, this bit should be in the `bridged` NIC section not the `bridge` network section, good spot.